### PR TITLE
Add i18n & router

### PR DESCRIPTION
**Pull request by HyTekCoop**
Add i18n & router and also delete empty files

### CODE REVIEW

File: src/application/router/router.ts
Comment (neSpecc) [inline src/application/router/router.ts:1]: lets call the file `index.ts`

**Review by neSpecc — COMMENTED** [review]
seems like i18 and router are not connected to the `src/index.ts`

File: src/application/i18n/index.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
 import Vue from 'vue';
 import VueI18n from 'vue-i18n';
-import languages from './languages/en'
+import en from './messages/en.json';
 
-Vue.use(VueI18n);
\ No newline at end of file
+Vue.use(VueI18n);
```
Comment (slaveeks) [inline src/application/i18n/index.ts:5]: I think, you need to call it in `index.ts` file

**Review by neSpecc — CHANGES_REQUESTED** [review]

File: src/application/router/index.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { createRouter, createWebHistory } from 'vue-router';
+
+const router = createRouter({
+  history: createWebHistory(),
+  routes: [
+    {
+      path: '/',
+      redirect: '/app',
```
[the remark this answers is not in the document]
Comment (HyTekCoop) [reply src/application/router/index.ts]: changes added

**Review by neSpecc — APPROVED** [review]

**Review by slaveeks — APPROVED** [review]

**Pull request merged**